lightning/src/ln/channelmanager.rs

```diff
@@ -1131,6 +1131,36 @@ impl ChannelDetails {
 	}
 }
 
+/// Used by [`ChannelManager::list_recent_payments`] to express the status of recent payments.
+/// These include payments that have yet to find a successful path, or have unresolved HTLCs.
+#[derive(Debug, PartialEq)]
+pub enum RecentPaymentDetails {
+	/// When a payment is still being sent and awaiting successful delivery.
+	Pending {
+		/// Hash of the payment that is currently being sent but has yet to be fulfilled or
+		/// abandoned.
+		payment_hash: PaymentHash,
+		/// Total amount (in msat) across all paths for this payment, not just the amount currently
+		/// inflight.
+		total_msat: u64,
```

Comment:
> When it says "all paths" is that said to account for MPP? Like if there were 2 paths for this payment, total_msat is the sum of the value received across both those paths?

Yup!

> Also I think it would maybe be helpful to add that this doesn't include fees? May need to double check, but from what I see in PendingOutboundPayment::Retryable it doesn't seem to(?). Since this is a sort of simpler interface on top of PendingOutboundPayment and because it lacks the other fee related fields that PendingOutboundPayment has (that might help a user infer whether total_msat includes fees), I figured I'd ask if it might be helpful to include in docs.

I think that's valid feedback, thanks for pointing it out -- I'll add it :)